lib/Agrammon/Model.pm6

```diff
@@ -379,4 +380,34 @@ class Agrammon::Model {
         });
         %$!distribution-map-cache
     }
+
+    # Create LaTeX docu
+    method create-docu( :%technical! ) {
```

Comment:
Is `docu` a LaTeX term, or just being too lazy to type `document`? BTW, if you use `#|` comments instead on routines, then they are doc comments.

Reply:
Also, I'm a bit uncomfortable with this being on `Model`. The model is, well, the core data model of Agrammon; formatting it into a LaTeX description feels like a non-core concept.

Reply:
> Is `docu` a LaTeX term, or just being too lazy to type `document`? BTW, if you use `#|` comments instead on routines, then they are doc comments.

Laziness .. renamed ...

Reply:
> Also, I'm a bit uncomfortable with this being on `Model`. The model is, well, the core data model of Agrammon; formatting it into a LaTeX description feels like a non-core concept.

... and moved into its own module.